Gemfile

```diff
@@ -20,9 +20,6 @@ group :development, :test do
   gem "decidim-dev", path: "."
 
   gem "brakeman", "~> 6.1"
-  gem "net-imap", "~> 0.2.3"
```

Comment:
We have a warning for multiple inclusions in these packages:

* net-imap
* net-pop
* net-smtp

The warning is:

> Your Gemfile lists the gem net-imap (~> 0.2.3) more than once.
> You should probably keep only one of them.
> Remove any duplicate entries and specify the gem only once.
> While it's not a problem now, it could cause errors if you change
> the version of one of them later.